Bottom: 707a09faefc0ab9f4339d4d2d9de51c28d1ba7a3
Top:    0bf9fb75bf0be42a634416e9c949a5b1a57f7e1e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-11 19:09:40 +0000

Refresh of remove-settings-init_done

---

diff --git a/src/global.h b/src/global.h
index 3794448..e7e7fa8 100644
--- a/src/global.h
+++ b/src/global.h
@@ -668,8 +668,7 @@ typedef struct			/* some resources... */
     /* connections is done */
     AutoPlace;			/* flag which says we should force placement of the
 				   windows on startup */
-  int HistorySize,		/* FIXME? Used in hid/xaw code only. */
-    init_done;
+  int HistorySize;		/* FIXME? Used in hid/xaw code only. */
 }
 SettingType, *SettingTypePtr;
